app/test_main.py

```diff
@@ -1,3 +1,16 @@
 from app.main import get_human_age
 
-# write your code here
+def test_should_return_zero_if_cat_and_dog_age_equal_14():
+    assert get_human_age(14, 14) == [0, 0]
```

Comment:
The test case `test_should_return_zero_if_cat_and_dog_age_equal_14` seems to be testing the same scenario as `test_should_return_extra_human_year_after_24_animal_years` but with different input values. Ensure that each test case is unique and tests a different scenario. This relates to the checklist item: 'Don't Repeat Yourself'.